### ISSUE TITLE
Expanded option

### ISSUE THREAD

**Issue by iPublicis** [apjanke/agnosterj-zsh-theme#19]
For a specific user it is needed to have full expanded directory always, even when on directories below the user's home directory.

Your `full` Style doesn't do that, I think.

How to had an `expand`  option that produces an output like `/home/john/Documents`instead of `~/Documents`?

**Comment by apjanke** [comment]
You are correct: `full` does not affect the abbreviation of the home directory to `~`. I can add something to control this.

Can you clarify how you'd like this to work? Do you mean:
* You know someone who wants to turn on "expand" for their Agnoster-J use
or:
* You have multiple user accounts/logins that you use, and you want to turn on "expand" for some of them, on a per-account basis?

**Comment by iPublicis** [comment]
@apjanke the first situation. I know someone that due to specific conditions needs to have the ~ expanded always.

**Comment by apjanke** [comment]
I have added a `$AGNOSTER_ABBREVIATE_HOME_DIR` option in https://github.com/apjanke/agnosterj-zsh-theme/commit/7394aee5d48a22c12d4fcdd3c22a1d9c73f6447f that you can set to `0` to turn off this abbreviation behavior. It works with any of the path styles (though it has weird interaction with the `shrink` style). Give it a try!

**Comment by iPublicis** [comment]
Thank you. I forwarded this to the user so he can test it. Asked him to do further feedback to you.

**Comment by apjanke** [comment]
No problem. Happy to make this useful for more people!
